libraries/chain/include/eosio/chain/abi_serializer.hpp

```diff
@@ -746,20 +756,10 @@ namespace impl {
                     "Failed to deserialize data for ${account}:${name}", ("account", act.account)("name", act.name));
       }
 
-      template<typename Resolver>
-      static void extract( const variant& v, signed_block& sb, Resolver resolver, abi_traverse_context& ctx )
-      {
-         signed_block_v0 v0;
-         extract( v, v0, std::move( resolver ), ctx );
-         sb = signed_block( v0, true );
-      }
-
       template<typename Resolver>
       static void extract( const variant& v, packed_transaction& ptrx, Resolver resolver, abi_traverse_context& ctx )
```

Comment:
This should now just be removed.

Reply:
You could potentially still make it to this via `variant_to_binary`, but I'm fine with removing it.